common/src/main/scala/uk/ac/wellcome/models/License.scala

```diff
@@ -2,10 +2,46 @@ package uk.ac.wellcome.models
 
 import com.fasterxml.jackson.annotation.JsonProperty
 
-case class License(
-  val licenseType: String,
-  val label: String,
+trait License {
+  val licenseType: String
+  val label: String
   val url: String
-) {
   @JsonProperty("type") val ontologyType: String = "License"
 }
+
+// ======================= AUTOGENERATED CODE STARTS ======================= //
```

Comment:
Wrong! Everything in this file is autogenerated – this should go right at the top.

Also, this comment should really exclude this code from scalafmt.